zcash_client_sqlite/src/lib.rs

```diff
@@ -426,9 +503,12 @@ impl<'a, P: consensus::Parameters> WalletWrite for DataConnStmtCache<'a, P> {
 
                 // Mark notes as spent and remove them from the scanning cache
                 for spend in &tx.shielded_spends {
-                    wallet::mark_spent(up, tx_row, &spend.nf)?;
+                    wallet::mark_sapling_note_spent(up, tx_row, &spend.nf)?;
                 }
 
+                //TODO
+                //wallet::mark_transparent_utxo_spent(up, tx_ref, &utxo.outpoint)?;
```

Comment:
Why is this a TODO?

Reply:
This will require an update to lightwalletd's CompactBlock format to implement properly. I think that what this means is that at present, we may not correctly detect when transparent UTXOs have been spent when recovering from seed backup. Opened https://github.com/zcash/librustzcash/issues/484

Reply:
And closed #484; this should just be deleted instead, since it's up to the client to mark UTXOs spent with data obtained from querying lightwalletd directly; the CompactBlock format will not include this data.